### PR TITLE
added section about master key config

### PR DESCRIPTION
**Pull request by mtrezza**

### CODE REVIEW

File: _includes/cloudcode/cloud-code-advanced.md
Comment (TomWFox) [inline _includes/cloudcode/cloud-code-advanced.md]:
```suggestion
By default, Parse Config parameters can be publicly read which may be undesired if the parameter contains sensitive information that should not be exposed to clients. A parameter can be made readable only with the master key by setting the `Master Key Only` property via the Parse Dashboard to `true`. 
```

**Review by TomWFox — CHANGES_REQUESTED** [review]
Wow fast, I love when the docs are ready even before the feature has been merged or released!

I think a bit of rewording is needed here, we'll wait to merge this until the relevant PRs have been merged and releases made.

**Comment by mtrezza** [comment]
Requires https://github.com/parse-community/parse-server/pull/5954 and https://github.com/parse-community/parse-dashboard/pull/1233 to be released.

**Review by TomWFox — APPROVED** [review]

**Review by TomWFox — APPROVED** [review]

**Comment by mtrezza** [comment]
Don't merge yet, I think some changes need to be made in the JS SDK as well. Discussion in https://github.com/parse-community/parse-server/issues/5930

**Comment by mtrezza** [comment]
I've added docs for https://github.com/parse-community/Parse-SDK-JS/pull/910 after changes have been approved. Will that be merged as well or is there another approval round needed?

**Review by TomWFox — APPROVED** [review]
LGMT!

**Comment by mtrezza** [comment]
Oops, I just deleted the branch by mistake. What should I do now, open a new PR?

**Comment by mtrezza** [comment]
Ah, it seems to be fine after reopening.

**Review by davimacedo — APPROVED** [review]
LGTM!

**Pull request merged**
